packages/breadcrumbs/src/styled/StyledChevronIcon.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import ChevronRightStrokeIcon from '@zendeskgarden/svg-icons/src/12/chevron-right-stroke.svg';
+
+const LTR_COMPONENT_ID = 'breadcrumbs.chevron_right_stroke_icon';
+const RTL_COMPONENT_ID = 'breadcrumbs.chevron_left_stroke_icon';
+
+const getGardenId = (rtl: boolean) => (rtl ? RTL_COMPONENT_ID : LTR_COMPONENT_ID);
+
+/**
+ * Accepts all `<svg>` props
+ */
+export const StyledChevronIcon = styled(ChevronRightStrokeIcon).attrs(props => ({
+  'data-garden-id': `${getGardenId(props.theme.rtl)}`,
+  'data-garden-version': PACKAGE_VERSION,
+  'aria-hidden': 'true'
+}))`
+  ${props => props.theme.rtl && `transform:rotate(180deg);`}
+  margin: 0 calc(8 / 14 * 1em);
```

Comment:
This is the right calculation from CSS, but it should be reasoned as a function based on theming values. Based on what we see in the designs, it is safe to assume the intent is a `4px` horizontal margin – so simply base it on `space.base`:

`em(props.theme.space.base, props.theme.fontSizes.md)` (`em` is a function from polished)

Also, there should be (almost) no `calc` in our CSS.

Reply:
> Also, there should be (almost) no calc in our CSS.

Thanks for pointing this unreasoned line of CSS out. I had a feeling I was cutting corners. My apologies and will make sure to be more thorough about hard coding magical values. I have updated this particular line of CSS to:

```jsx
margin: 0 ${props => em(props.theme.space.base * 2, props.theme.fontSizes.md)};
```